irr/src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+add_executable(tests
+	testCGLTFMeshFileLoader.cpp
+	"${PROJECT_SOURCE_DIR}/source/Irrlicht/CReadFile.cpp"
+)
+
+set_target_properties(tests PROPERTIES
+	CXX_STANDARD 11
```

Comment:
why can't this reuse the settings from he parent project?

Reply:
It sure can, I should not have set this here, as it may become out of sync with the project standard.